operations/helm/charts/mimir-distributed/values.yaml

```
@@ -505,6 +505,13 @@ rbac:
     seccompProfile:
       type: RuntimeDefault
 
+# -- KEDA Autoscaling configuration
+kedaAutoscaling:
+  # If empty, the helm chart uses the metamonitoring URL
```

```suggestion
  # -- A Prometheus-compatible URL. Cadvisor and Mimir metrics for the Mimir pods are expected in this server. For more information on the required metrics see [Monitor system health](https://grafana.com/docs/helm-charts/mimir-distributed/latest/run-production-environment-with-helm/monitor-system-health/) If empty, the helm chart uses the metamonitoring URL from metaMonitoring.grafanaAgent.metrics.remote.url. If that is empty, then the Mimir cluster is used.
```

adding a bit more docs around what's expected to enter here.